src/importexport/capella/internal/capella.cpp

```diff
@@ -843,12 +845,13 @@ static Fraction readCapVoice(Score* score, CapVoice* cvoice, int staffIdx, const
             ClefType nclef = o->clef();
             LOGD("%d:%d <Clef> %s line %d oct %d clef %d",
                  tick.ticks(), staffIdx, o->name(), int(o->line), int(o->oct), int(o->clef()));
-            if (nclef == ClefType::INVALID) {
+            if (nclef == ClefType::INVALID || nclef == pclef || nclef == score->staff(staffIdx)->defaultClefType().concertClef) {
```

Comment:
Okay, one more idea (last one, I promise!): initialise `pclef` to `pclef || nclef == score->staff(staffIdx)->defaultClefType().concertClef`, so that you can simplify this condition to 
```suggestion
            if (nclef == ClefType::INVALID || nclef == pclef) {
```

Reply:
OK